nengo/networks/tests/test_product.py

```diff
@@ -35,3 +35,100 @@ def test_sine_waves(Simulator, plt, seed):
     plt.xlim(right=t[-1])
 
     assert rmse(AB[:len(offset), :], sim.data[p][offset, :]) < 0.2
+
+
+class HilbertCurve(object):
```

Comment:
I'm not sure if we need it right now, but I could see it being useful for other tests. Also, it seems kind of strange or at least surprising having it right in the middle of this test file.